### ISSUE TITLE
xpra fails to compile - missing resource

### ISSUE THREAD

**Issue by markjfisher** [ska-sa/homebrew-tap#12]
The xpra-0.10.9.tar.bz2 file is no longer at winswitch.org/src, so the brew fails to work.

```
==> Installing xpra
==> Downloading http://winswitch.org/src/xpra-0.10.9.tar.bz2

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "xpra"
Download failed: http://winswitch.org/src/xpra-0.10.9.tar.bz2
```

**Comment by markjfisher** [comment]
Also, with the version bump, it looks like it's been renamed to winswitch-0.12.20.src.tar.bz2 (this is the current "source download" link from http://winswitch.org/dev/#source

**Comment by ludwigschwardt** [comment]
Fixed in 1937c6d83d3c4d0d (a year later 😢 ).
